### PR TITLE
create rohansuri17.md

### PR DESCRIPTION
**Pull request by rohansuri17**
Fixes N/A

### Description
Created my profile! Yay!
### Raw.Githack preview link
https://raw.githack.com/rohansuri17/rohansuri17-OLE.github.io/rohan/#!pages/vi/profiles/rohansuri17.md

### CODE REVIEW

**Review by jayala-29 — DISMISSED** [review]
nice :)

**Review by law-lin — DISMISSED** [review]
Welcome!

File: pages/vi/profiles/rohansuri17.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+I have previous experience with software development, full stack web development, and software engineering.
+I am very interested in Artificial Intelligence and Machine Learning and spend a lot of free time reading up 
+on different AI/ML concepts. I am very well versed in software and am here to expand 
+my skill-set even more to include working with Raspberry PI. 
+
+<br>
+My favorite colors are:<br> 
+1. Red<br>
+2. Blue<br>
+3. Green<br>
```
Comment (dogi) [inline pages/vi/profiles/rohansuri17.md]: since this is a markdown exercise no html please

File: pages/vi/profiles/rohansuri17.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Rohan Suri #
+<br>
+Denver,CO MST 
+<br>
+Ubuntu 20.04 LTS
+<br>
```
Comment (dogi) [inline pages/vi/profiles/rohansuri17.md]: since this is a markdown exercise no html please

**Review by dogi — COMMENTED** [review]
nice start @rohansuri17 

please fix the comments
and add a maybe more complex markdown element
like link or picture

File: pages/vi/profiles/rohansuri17.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Rohan Suri #
+Denver,CO MST 
+Ubuntu 20.04 LTS
+
+![Profile Picture](https://github.com/rohansuri17/rohansuri17-OLE.github.io/blob/master/49506437_2602189069821948_6174767657032613888_n.jpg)
```
Comment (dogi) [inline pages/vi/profiles/rohansuri17.md]: @rohansuri17 think it is better to add this picture to the repo ...

Comment (rohansuri17) [reply pages/vi/profiles/rohansuri17.md:5]: Looks good to me!

**Review by dogi — APPROVED** [review]

**Pull request merged**